pkg/session/manager.go

```diff
@@ -23,17 +24,25 @@ type Manager struct {
 	clientID        string
 }
 
+type CallbackResult struct {
+	SessionID   string
+	CSRFToken   string
+	RedirectURI string
+}
+
 func NewManager(
 	oidc oidc.ProviderRepository,
 	sessions Repository,
 	auditLogger *otlpaudit.AuditLogger,
+	pkce pkce.Source,
```

Comment:
It isn't necessarily since `pkce.Source` was used by value, not by pointer. It's better to avoid irrelevant changes in the PR.